### PR TITLE
CLIPLAYEREX-120 Size of dynamic thumbstick area in landscape mode

### PR DESCRIPTION
**Pull request by JHelmsAtROBLOX**
The gesture area for thumbstick and camera is now a different proportion of the screen in landscape vs. portrait mode. (The thumbstick take up 2/5 of the screen in portrait mode and 1/2 in landscape mode.)

### CODE REVIEW

File: PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx [hunk 1]
```diff
@@ -140,9 +124,9 @@ do
 			GestureArea.BackgroundTransparency = 1.0
 			GestureArea.Visible = true
 			GestureArea.BackgroundColor3 = Color3.fromRGB(0, 0, 0)
-			GestureArea.Size = UDim2.new(1, 0, .6, 0)
-			GestureArea.Position = UDim2.new(0, 0, 0, 0)
 			GestureArea.Parent = ScreenGui
+			layoutGestureArea(PortraitMode)
```
Comment (hockeychaos) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx]: we should layout before parenting, causes less signals to be fired to rendering. Not a huge deal though.

File: PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx [hunk 2]
```diff
@@ -169,6 +153,36 @@ do
 			OnCharacterAdded(character)
 		end)
 	end
+	
+	-- Check for changes in ViewportSize to decide if PortraitMode
+	local CameraChangedConn = nil
+	local function onWorkspaceChanged(property)
```
Comment (hockeychaos) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx]: should we check to see if we have touch input? or does this work w/o touch input?

Comment (JHelmsAtROBLOX) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx]: Requires TouchInput, will fix

Comment (hockeychaos) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx]: @JHelmsAtROBLOX any updates on this

Comment (JHelmsAtROBLOX) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx]: I could have sworn I replied to your comments and submitted another commit! Sorry, let me see where I left this...

Comment (JHelmsAtROBLOX) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx]: Forgot to push the commit :(, here you go
